### ISSUE TITLE
glm Binomial with 3-column endog BUG?

### ISSUE THREAD

**Issue by josef-pkt** [statsmodels/statsmodels#83]
see also, and example from https://github.com/statsmodels/statsmodels/issues/26

According to the description of glm Binomial it is possible to define
the endog as a 2d (nobs, 2) array with (success, failures) as columns.

The example with 3 columns in endog actually worked without raising a
ValueError. The example has only 0s and 1s, a dummy encoding with
np.all(y.sum(1) == 1).

The result with 3 endog columns is the same as (nobs,2) endog with the
first column and the (max of the 2nd and 3rd columns) as second
column.
(I only checked params and bse)

I guess this wasn't intended.

(since there are no attachments on github issues)

```
# -*- coding: utf-8 -*-
"""
Created on Wed Oct 05 22:29:28 2011

@author: josef
"""

import numpy as np
import scikits.statsmodels.api as sm

np.random.seed(9876345)

y = np.zeros((700, 3))
x = np.random.randn(700, 10)
y[np.arange(700),np.random.random_integers(3,size=700)-1] = 1
res1 = sm.GLM(y, x, family=sm.families.family.Binomial()).fit()
print 'Y.shape', y.shape
print 'res1.params'
print res1.params
print 're1.bse'
print res1.bse
print

y2 = np.column_stack((y[:,0], y[:,1:].max(1)))
res2 = sm.GLM(y2, x, family=sm.families.family.Binomial()).fit()
print 'y2.shape', y2.shape
print 'res2.params'
print res2.params
print 're1.bse'
print res2.bse

'''
result

Y.shape (700, 3)
res1.params
[ 0.04970902 -0.03642792  0.02073708 -0.0085053  -0.18709573  0.04136817
 -0.1112802  -0.04967136 -0.05388642  0.02801581]
re1.bse
[ 0.07703376  0.07501304  0.07644803  0.07453469  0.07548221  0.07512962
  0.07694712  0.07581411  0.07264488  0.07719404]

y2.shape (700, 2)
res2.params
[ 0.04970902 -0.03642792  0.02073708 -0.0085053  -0.18709573  0.04136817
 -0.1112802  -0.04967136 -0.05388642  0.02801581]
re1.bse
[ 0.07703376  0.07501304  0.07644803  0.07453469  0.07548221  0.07512962
  0.07694712  0.07581411  0.07264488  0.07719404]
'''
```

**Comment by josef-pkt** [comment]
bump for 0.10, so it doesn't get lost  (I just found it by chance with an unrelated search)

needs checking

**Comment by jseabold** [comment]
This was closed by #6017
